tests-manual/box.R

```diff
@@ -1,23 +1,25 @@
 # A dashboard body with a row of infoBoxes and valueBoxes, and two rows of other boxes
 
 library(shiny)
+library(shinydashboard)
+
 body <- dashboardBody(
 
-  # infoBoxes
+  # infoBoxes -> first row
   fluidRow(
     infoBox(
-      "Orders", uiOutput("orderNum2"), "Subtitle", icon = icon("credit-card")
+      "Orders", uiOutput("orderNum2"), subtitle = "Test", icon = icon("credit-card"),
+      fill = T, progressValue = uiOutput("progressBarValue")
```

Comment:
Also, I think it wouldn't really make sense for users to be able to enter in a numeric value when the value is static, but have to manually create a div for dynamic output here. Probably the best thing for users to do is dynamically output the entire `infoBox`. For this particular example, there's probably no need to make the `progressValue` dynamic, since no one will actually use it this way.